contributing/development/compiling/compiling_for_linuxbsd.rst

```diff
@@ -16,7 +16,7 @@ Requirements
 For compiling under Linux or other Unix variants, the following is
 required:
 
-- GCC 9+ or Clang 6+.
+- GCC 13+ or Clang 6+.
```

Comment:
I don't think we should change the minimum supported version due to a bug we caused. We should just fix that bug.

Reply:
Sounds good to me